src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -116,6 +116,7 @@ or PlaybackStateCompat.StateSkippingToQueueItem
 		{
 			MediaElement.Duration = TimeSpan.FromMilliseconds(Player.Duration < 0 ? 0 : Player.Duration);
 			MediaElement.Position = TimeSpan.FromMilliseconds(Player.CurrentPosition < 0 ? 0: Player.CurrentPosition);
+			MediaElement.Position = TimeSpan.FromMilliseconds(Player.CurrentPosition < 0 ? 0 : Player.CurrentPosition);
```

Comment:
Looks like this is a duplicate

Reply:
Thanks Vlad - good catch! 

This was a user-error after doing `dotnet format`.